src/views/PostView.vue

```diff
@@ -50,12 +50,12 @@ const loadMarkdown = () => {
   }
   const title = decodeURIComponent(currentRoute.value.params.name.toString())
 
-  const module = modules.value.find(item => item.meta.title === title && item.meta.seriesTitle === sTitle)
-
+  const module = modules.value.find(
+    (item) => item.meta.title === title && item.meta.seriesTitle === sTitle
+  )
```

Comment:
Change is only formatting. Nice to have but in separate PR. Prevent code format changes and it will help reviewers focus on the main code.